sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/CosmosEncryptionAsyncContainer.java

```diff
@@ -1049,72 +1125,88 @@ private <T> Function<CosmosPagedFluxOptions, Flux<FeedResponse<T>>> transformQue
         Transformer<T> transformer,
```

Comment:
Not introduced by this PR but `pagedFluxOptions` is unused and can be removed.

Reply:
Fixed